webpack/webpackPlugin.ts

```diff
@@ -126,7 +133,7 @@ class ScoobieWebpackPlugin {
      * {@link https://webpack.js.org/guides/asset-modules/}
      */
     const rules = compiler.options.module.rules.filter(
-      (rule) => !ruleTestsToRemove.has(String(rule.test)),
+      (rule) => rule === '...' || !ruleTestsToRemove.has(String(rule.test)),
```

Comment:
This should be the only logic change. Not sure what `...` means in this context.